group_vars/all

```diff
@@ -351,3 +351,8 @@
   gpfs_groups:
     - {"name": "gpfs4", "authorized_keys":"/gpfs4/data/user/home/$DOWNSTREAM_USER/.ssh/authorized_keys", "host": "login001", "private_key":"/gpfs4/data/user/home/$DOWNSTREAM_USER/.ssh/id_ecdsa"}
     - {"name": "gpfs5", "authorized_keys":"/gpfs5/data/user/home/$DOWNSTREAM_USER/.ssh/authorized_keys", "host": "login002", "private_key":"/gpfs5/data/user/home/$DOWNSTREAM_USER/.ssh/id_ecdsa"}
+
+# job_submit_plugin
+  enable_job_submit_plugin: true
+  job_submit_lua_url: "https://gitlab.rc.uab.edu/rc/slurm-factory/-/blob/main/etc/slurm/job_submit.lua"
```

Comment:
Use this link instead:
https://gitlab.rc.uab.edu/rc/slurm-factory/-/raw/main/etc/slurm/job_submit.lua

Reply:
fixed ca467f585b472f86d47b568ff2764ebbbe1e616b